omspy/order.py

```diff
@@ -627,7 +627,7 @@ def update_ltp(self, last_price: Dict[str, float]):
             order.update_ltp(last_price)
         return self.ltp
 
-    def update_orders(self, data: Dict[str, Dict[str, Any]]) -> Dict[str, bool]:
```

Comment:
You can push this change
but I need to rewrite so that the return type is `Dict[str,bool]`

Reply:
that's fine